outputs.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os/exec"
+	"strings"
+
+	"github.com/bitrise-io/go-utils/log"
+)
+
+/// The response from Slack POST
+type SendMessageResponse struct {
+	/// The Thread Timestamp
+	Timestamp string `json:"ts"`
+}
+
+/// Export the output variables after a successful response
+func exportOutputs(conf *Config, resp *http.Response) error {
+
+	if !isRequestingOutput(conf) {
+		log.Debugf("Not requesting any outputs")
+		return nil
+	}
+
+	is_webhook := strings.TrimSpace(selectValue(string(conf.WebhookURL), string(conf.WebhookURLOnError))) != ""
```

Comment:
The linter is asking that this variable be renamed to be camel case, like:

```suggestion
	isWebhook := strings.TrimSpace(selectValue(string(conf.WebhookURL), string(conf.WebhookURLOnError))) != ""
```